appveyor.yml

```diff
@@ -39,3 +39,8 @@ build: off
 
```

Comment:
I don't know what is going on with appveyor but it was not finding already the compile rule for the jamfile

ERROR: rule "compile" unknown in module "Jamfile<C:\projects\boost-root\libs\ratio\test>".

This is weird.

Reply:
from the commit history it seems that the appveyor script was broken since the ci script was originally added.
I'm unfortunately not a b2 nor appveyor expert, otherwise I'd have tried to fix it on the go.